modular_skyrat/modules/black_mesa/code/mobs/nihilanth.dm

```diff
@@ -66,28 +66,7 @@
 
 /mob/living/simple_animal/hostile/blackmesa/xen/nihilanth/death(gibbed)
 	. = ..()
```

Comment:
All this proc does is call parent - does it need to be kept?

Reply:
Yes, or else the Nihilanth doesn't get gibbed.